server/auth/types/basic/routes.ts

```diff
@@ -194,8 +197,12 @@ export class BasicAuthRoutes {
             sessionStorage.tenant = selectTenant;
           }
           this.sessionStorageFactory.asScoped(request).set(sessionStorage);
-
-          return response.ok({
+          return response.redirected({
+            headers: {
+              location: `${this.coreSetup.http.basePath.serverBasePath}${
+                request.url.path?.split('?')[1]
+              }`,
+            },
             body: {
```

Comment:
Does it make sense to have body for a 302 redirection?

Reply:
will remove it